tests/test-core.lisp

```diff
@@ -149,3 +152,30 @@
 		  #xAA #xBB #xCC #xDD #xEE #xFF   ;; 16th repetition
 		  #x01 #x02 #x03 #x04 #x05 #x06)) ;; SecureOn password
 	"encode-payload matches for aa-bb-cc-dd-ee-ff with SecureOn password")))
+
+(deftest make-magic-packet
+  (testing "test make-magic-packet with vectors"
+    (let ((items (list (list :addr (make-octet-vector '(0 0 0 0 0 0))
+			     :octets #(0 0 0 0 0 0)
```

Comment:
This is not an octet vector, this is a simple-vector - and that's good, because you aren't looking for 100% code efficiency here, it makes sense to support both.

There's no mention of that in the README though - I think the API exported from the API page should have its own README section. (But then, I'm asking you to document stuff, which is a tough thing to do in the Lisp world. :D)

Reply:
README updated to indicate that `(simple-array (unsigned-byte 8) (6))` vectors are also supported.

I like the idea of having a separate README for the API documentation. I'll see if I find some spare time to do that as well :)